src/forms-opportunities/oppty-handlers/accessibility-handler.js

```diff
@@ -463,6 +468,18 @@ export default async function handler(message, context) {
     // Create individual suggestions from Mystique data
     await createFormAccessibilitySuggestionsFromMystique(a11y, opportunity, context);
 
+    // send message to importer for code-fix generation
+    const isAutoFixEnabled = await isAuditEnabledForSite(`${opportunity.getType()}-auto-fix`, site, context);
```

Comment:
for forms it will be `forms-opportunity-auto-fix`. If that is the case we should think of breaking it for experimental opportunity.

Reply:
For form accessibility it will be `form-accessibility-auto-fix`